dataactbroker/handlers/fileHandler.py

```diff
@@ -1179,3 +1145,55 @@ def get_error_metrics(submission):
     except Exception as e:
         # Unexpected exception, this is a 500 server error
         return JsonResponse.error(e,StatusCode.INTERNAL_ERROR)
+
+
+def list_submissions(page, limit, certified):
+    """ List submission based on current page and amount to display. If provided, filter based on
+    certification status """
+    sess = GlobalDB.db().session
+
+    offset = limit*(page-1)
+
+    cgac_codes = [aff.cgac.cgac_code for aff in g.user.affiliations]
+    query = sess.query(Submission).\
+        filter(sa.or_(Submission.cgac_code.in_(cgac_codes),
+                      Submission.user_id == g.user.user_id))
+    if certified != 'mixed':
+        query = query.filter_by(publishable=certified)
+    submissions = query.order_by(Submission.updated_at.desc()).\
+            limit(limit).offset(offset)
+
+    return JsonResponse.create(StatusCode.OK, {
+        "submissions": [serialize_submission(s) for s in submissions],
+        "total": query.count()
+    })
```

Comment:
I realize we aren't doing this right now, but should we be accounting for website_admin in this filter? Can they see all submissions? If so, it doesn't seem to be reflected here